manage/scheduler.rst

```diff
@@ -3,61 +3,83 @@ Scheduler
 
 The scheduler performs time-based automated actions. You can set up your own
 schedulers, configure at which points in time they should run, set up conditions
-to determine which tickets they should affect, and then configure the actions
-that you want to be executed on these tickets.
-
-.. note::
-
-   Schedulers with *Action: Delete* are currently the only way in the Zammad
-   front end to permanently delete tickets. This limitation is intentional as
-   Zammad is designed to be revision-proof. A possible use case for such a
-   scheduler is to delete spam tickets some time after creation (e.g. 30 days).
+to determine which objects they should affect, and then configure the actions
+that you want to be executed on these objects.
 
 .. warning::
 
    While it is possible to delegate scheduler permissions to normal agents with
-   the ``admin/scheduler`` permission, it is inadvisable to do so. Malicious
+   the ``admin.scheduler`` permission, it is inadvisable to do so. Malicious
    agents could use a scheduler to access tickets in restricted groups
-   (by moving them to a non-restricted group) or to delete arbitrary tickets.
+   (by moving them to a non-restricted group) or to arbitrarily delete tickets.
 
 .. hint::
 
-   | Schedulers can be used to send periodic reminder emails.
-     Use :doc:`/system/variables` to build highly-customized email templates.
-   |
-   | Schedulers only perform ``2000`` tickets per run. This is a security
-     function in case you accidentally miss configure the scheduler.
+   Schedulers can be used to send periodic reminder emails.
+   Use :doc:`/system/variables` to build highly-customized email templates.
+
+   Schedulers only perform ``2000`` objects per run. This is a security
+   function in case you accidentally misconfigured the scheduler.
 
 Add a new scheduler
 ===================
 
+Create a new scheduler job by clicking on the "New scheduler" button in the top
+right corner. Then Zammad presents a dialogue where you can create your own one.
```

Comment:
`dialogue` => `dialog`

_Dialogue_ is mostly used for conversations, while _dialog_ is the modal element we all learned to love :)